search-enterprise/latest/config/index.md

```diff
@@ -116,7 +116,10 @@ The tool may be used as a standalone jar. The table below lists the main configu
 | alfresco.reindex.metadataIndexingEnabled | Reindex document metadata | true |
 | alfresco.reindex.contentIndexingEnabled | Reindex document content | true |
 | alfresco.reindex.pathIndexingEnabled | Reindex document Path property | false |
-    
+| spring.activemq.username | ActiveMQ username | admin |
+| spring.activemq.password | ActiveMQ password | admin |
+| spring.batch.datasource.username | Username for the in-memory database created and used by the re-indexing app | SA |
```

Comment:
is not totally correct, for the horizontal scalability we use those properties to set up the connection to a shared database like postgresql. Should we document also other properties required in the horizontal scalability configuration? I listed them below:
  - spring.batch.datasource.driver-class-name
  - spring.batch.drop.script
  - spring.batch.schema.script

Reply:
Thanks - I have added these properties and updated the existing description.